package.json

```diff
@@ -27,6 +28,7 @@
     "luxon": "^2.3.0",
     "mitt": "^2.1.0",
     "register-service-worker": "^1.7.1",
+    "secure-ls": "^1.2.6",
```

Comment:
Sir, it's coming here as diff because the dependency has been removed from the package.json in one of the currently merged PR but this PR is not updated with main branch.